components/monitoring/prometheus/base/prometheus-servicemonitors.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   endpoints:
   - bearerTokenSecret:
-      name: "prometheus-k8s-token-xhrjb"
+      name: "prometheus-k8s-token-md9md"
       key: token
```

Comment:
Let change it back to `prometheus-k8s-token-xhrjb` as that is needed for the current deployment in stage

Reply:
my apologies, I had already noticed that unneeded change and reverted it but I suspect I forgot to push it earlier 🤷‍♂️

Reply:
this PR is aleady doing that.